test/test_models.py

```diff
@@ -17,9 +18,10 @@ def test_path(swagger_20):
 @pytest.mark.parametrize("override", (False, True))
 @pytest.mark.parametrize("converter", (lambda x: x, lambda x: x + "/"))
 def test_as_requests_kwargs(override, server, base_url, swagger_20, converter):
+    headers = {"User-Agent": USER_AGENT}
     base_url = converter(base_url)
     endpoint = Endpoint("/success", "GET", {}, swagger_20)
-    kwargs = {"endpoint": endpoint, "cookies": {"TOKEN": "secret"}}
+    kwargs = {"endpoint": endpoint, "cookies": {"TOKEN": "secret"}, "headers": headers}
```

Comment:
I'd rather revert this change and verify that by default headers are updated with the "User-Agent" header

Reply:
as_requests_kwargs dont set default user agent